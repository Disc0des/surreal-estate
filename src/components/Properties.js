import React from "react";
import PropertyCard from "./PropertyCard";
import "../styles/Properties.css";

function Properties() {
  return (
    <div className="property-card-cont">
      <PropertyCard
        title="nice house"
        type="flat"
        bedrooms="2"
        bathrooms="2"
        city="Manchester"
        price="Â£100000"
        email="joe@surreal"
      />
    </div>
  );
}

export default Properties;
