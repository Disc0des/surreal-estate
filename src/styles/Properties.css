.property-card-cont {
  background: url("../page-background.jpeg");
  background-size: cover;
  background-position-x: center;
  background-repeat: none;
  height: 100vh;
  width: 100vw;
  display: flex;
  flex-wrap: wrap;
  align-items: center;
  justify-content: center;
  overflow-y: scroll;
  scroll-behavior: smooth;
}

.property-card-cont .alert-error {
  align-self: baseline;
}

