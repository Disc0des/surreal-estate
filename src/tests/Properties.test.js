import React from "react";
import { render } from "@testing-library/react";
import Properties from "../components/Properties";

describe("View Properties page", () => {
  it("renders property cards container when GET request returns results", () => {
    const { getByText } = render(<Properties />);
    expect(getByText()).toMatchSnapshot();
  });
});
