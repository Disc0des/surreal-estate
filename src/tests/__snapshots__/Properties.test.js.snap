// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`View Properties page renders property cards container when GET request returns results 1`] = `
<DocumentFragment>
  <div
    class="property-card-cont"
  />
</DocumentFragment>
`;
